docs/gradle-plugin/index.md

```diff
@@ -241,6 +249,16 @@ Example:
 ```
 
 
+## HTML report description
+![Example of HTML report](html.png)
+- green indicates a line that has been executed at least once (covered)
+- red indicates a line that has never been executed (missed)
+- yellow indicates a line in which there is a branching, but at least one of the branches was not executed (partially covered)
+- code that was excluded from the report using filters is not highlighted in any color
+
+In the HTML report, Kover shows coverage by lines, it cannot display detailed coverage for an expression or separately for each code branch on a line.
```

Comment:
This is the limitation of default Kover tool, but Jacoco can do this?

Reply:
No, JaCoCo also can't